client/src/components/carousel.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import $ from 'jquery';
+
+
+class Carousel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    };
+  }
+
+  componentDidMount() {
+    this.getRequest();
+  }
+
+  getRequest() {
```

Comment:
Will this getRequest method ever be used outside of componentDidMount()? If not, maybe move it inside the CDM.

Reply:
Not 100% sure about what the best practices is here, but I would prefer to keep this separated as I believe it is easier to read.